src/Httpful/Response/Headers.php

```diff
@@ -36,7 +46,7 @@ public static function fromString($string)
      */
     public function offsetExists($offset)
     {
-        return isset($this->headers[strtolower($offset)]);
```

Comment:
It's been three years since the last mayor release, and about one year since the last release. I think it's time for some breaking changes :)